clib/mininet_test_base.py

```diff
@@ -812,6 +821,15 @@ def scapy_dhcp(self, mac, iface, count=1, dst=None):
                  dst, mac, IPV4_ETH),
             iface, count)
 
+    def scapy_icmp(self, mac, iface, count=1, dst=None):
+        if dst is None:
+            dst = 'ff:ff:ff:ff:ff:ff'
+        return self.scapy_template(
+            ('Ether(dst=\'%s\', src=\'%s\', type=%u) / '
+             'IP(src=\'0.0.0.0\', dst=\'255.255.255.255\') / ICMP() / '
```

Comment:
An IP src of 0.0.0.0 and dst of 255.255.255.255 doesn't really make sense here, typically that is only used for DHCP packets (port 67/68). Can we add an argument to this function that allows us to pass in the ip src and dst of the test hosts? (you could set the IP DST to the local broadcast address for the subnet if you want the packet to go to all hosts).